text/0036-ternary.md

```diff
@@ -0,0 +1,254 @@
++ Feature name: ternary
++ Start date: 2020-05-28
++ RFC PR: [iotaledger/bee-rfcs#36](https://github.com/iotaledger/bee-rfcs/pull/36)
++ Bee issue: [iotaledger/bee#80](https://github.com/iotaledger/bee/issues/80)
+
+# Summary
+
+This RFC introduces both the API and implementation of the `bee-ternary`
+crate, a general-purpose ternary manipulation crate written in Rust that may be
+used to handle ternary data in the IOTA ecosystem.
+
```

Comment:
I'd add a link to I'd add a link to https://docs.iota.org/docs/getting-started/0.1/introduction/ternary